backend/internal/authn/oidc/service.go

```diff
@@ -166,7 +166,7 @@ func (s *oidcAuthnService) ValidateIDToken(idpID, idToken string) *serviceerror.
 
 	// Validate ID token signature using JWKS endpoint if available
 	if oAuthClientConfig.OAuthEndpoints.JwksEndpoint != "" {
-		err := s.jwtService.VerifyJWTSignatureWithJWKS(idToken, oAuthClientConfig.OAuthEndpoints.JwksEndpoint)
+		err := s.jwtService.VerifyJWTWithJWKS(idToken, oAuthClientConfig.OAuthEndpoints.JwksEndpoint, "", "")
```

Comment:
Since we keep the `VerifyJWTSignatureWithJWKS()`, do we need to call `VerifyJWTWithJWKS()` with empty params here?

Reply:
Empty params are for the audience and issuer. If expected aud or issuer is not passed, it won't be validated. Added it this way since we don't know/ doesn't have a way to store the issuer/ audience values.

Reply:
It doesn't decode the token and validate the general claims such as the token expiry

Reply:
Ack.